app/resources/css-default.template

```diff
@@ -16,18 +16,17 @@
 
 /* All pages */
 
-html {
-  padding: 15px;
+.window {
```

Comment:
Can you move this below "body"? Roughly speaking, outer element or above element should come first.